dev/src/bulk-writer.ts

```diff
@@ -316,10 +316,14 @@ export class BulkWriter {
    * @private
    */
   private _errorFn: (error: BulkWriterError) => boolean = error => {
+    const isRstStreamError =
+      (error.code as number) === Status.INTERNAL &&
```

Comment:
1) We should be able to retry Status.INTERNAL regardless of whether this is a RST_STREAM error.
2) We should only add this retry for deletes.

Reply:
Would you please explain why the retry should be only for deletes?
The RST could happen on any other methods, this limitation comes from idempotency?

For example, Spanner automatically retries transaction commits, regardless of the queries within the txn. I wonder if there is a firestore specific thing here.